lib/DriverTool/swift_parse_test_main.cpp

```diff
@@ -87,19 +89,26 @@ struct LibParseExecutor {
     Parser parser(bufferID, *SF, /*SILParserState=*/nullptr);
     SmallVector<ASTNode> items;
     parser.parseTopLevelItems(items);
+
+    return llvm::Error::success();
   }
 };
 
 struct SwiftParserExecutor {
   constexpr static StringRef name = "SwiftParser";
 
-  static void performParse(llvm::MemoryBufferRef buffer, ExecuteOptions opts) {
+  static llvm::Error performParse(llvm::MemoryBufferRef buffer,
+                                  ExecuteOptions opts) {
 #if SWIFT_BUILD_SWIFT_SYNTAX
     // TODO: Implement 'ExecuteOptionFlag::SkipBodies'
     auto sourceFile = swift_ASTGen_parseSourceFile(
         buffer.getBufferStart(), buffer.getBufferSize(), /*moduleName=*/"",
         buffer.getBufferIdentifier().data(), /*ASTContext=*/nullptr);
     swift_ASTGen_destroySourceFile(sourceFile);
+    return llvm::Error::success();
+#else
+    return llvm::createStringError(llvm::inconvertibleErrorCode(),
+                                   "SwiftParser is not supported")
```

Comment:
@rintaro, this C++ syntax error [broke builds that don't use SwiftSyntax](https://ci-external.swift.org/job/oss-swift-RA-linux-ubuntu-16.04-android-arm64/5711/console):
```
/home/ubuntu/jenkins/workspace/oss-swift-RA-linux-ubuntu-16.04-android-arm64/swift/lib/DriverTool/swift_parse_test_main.cpp:111:67: error: expected ';' after return statement
                                   "SwiftParser is not supported")
```